host/hackrf-tools/src/hackrf_cpldjtag.c

```diff
@@ -31,23 +31,23 @@
 #include <stdint.h>
 
 #ifdef _MSC_VER
-#ifdef _WIN64
+	#ifdef _WIN64
 typedef int64_t ssize_t;
-#else
+	#else
 typedef int32_t ssize_t;
-#endif
+	#endif
 #endif
```

Comment:
This combination, of indenting the preprocessor statements but not the declarations between them, seems a bit odd.